cmd/node/main.go

```diff
@@ -488,6 +486,41 @@ func main() {
 		log.Infof("vochain chainID %s", srv.App.ChainID())
 	}
 
+	//
+	// Validator
+	//
+	if globalCfg.Mode == types.ModeMiner {
+		// create the key for the validator used to sign transactions
+		signer := ethereum.SignKeys{}
+		if err := signer.AddHexKey(globalCfg.Vochain.MinerKey); err != nil {
+			log.Fatal(err)
+		}
+		validator, err := srv.App.State.Validator(signer.Address(), true)
```

Comment:
I have doubts about this part.
Imagine a validator is set via consensus at block 100. When the new validator starts the syncing process from scratch, L#503 will be executed and the Keykeeper will not be initialized.
The validator could keep syncing until, at block 100, acknowledges that is a validator, but the node must be restarted to enable the Keykeeper.

Reply:
That's a good observation. 
However, we don't have currently the transaction logic implemented for adding new validators. Thus, this is not a real problem right now.
When we implement the validator add/del logic, we will need to enable a mechanism to trigger actions on the validator to start processing blocks. But that's not for now.